clients/imodels-client-authoring/src/operations/lock/LockOperationParams.ts

```diff
@@ -4,12 +4,17 @@
  *--------------------------------------------------------------------------------------------*/
 import { CollectionRequestParams, IModelScopedOperationParams } from "@itwin/imodels-client-management";
 
-import { LockedObjects } from "../../base/types";
+import { LockLevel, LockedObjects } from "../../base/types";
+
+/** Valid values for Lock filtering by their level. */
+export type LockLevelFilter = LockLevel.Shared | LockLevel.Exclusive;
```

Comment:
I'm not sure if it's better to explicitly define valid lock levels or use `Exclude<LockLevel, LockLevel.None>`. I favor exclusively defining it to avoid bugs in case `LockLevel` is added and easier readability.

Reply:
I prefer this approach too, closing this